### ISSUE TITLE
Installing `@knocklabs/react` using npm workspaces results in `EUNSUPPORTEDPROTOCOL`

### ISSUE THREAD

**Issue by hill** [knocklabs/javascript#71]
When installing `@knocklabs/react` using npm workspaces, I am getting `EUNSUPPORTEDPROTOCOL`

```
❯ npm i @knocklabs/react --workspace=frontend

npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "workspace:": workspace:^
```

We are unsure why this is occurring, and I believe we were previously able to install the packages correctly.

We are thinking this change last week might be involved? https://github.com/knocklabs/javascript/commit/7786ec5bd7b280e04e0dd304d7c8732211a41ed2#diff-1f344ac391eeecc21ec0f01fb07430a47f4b80d20485c125447d54c33c4bbfc4L53-R54

Would love any assistance!

Please let me know if you need further details from me :)

cc @sergioperezf

**Comment by connorlindsey** [comment]
Hey @hill, we've identified an issue with our publishing process and our getting a fix out for that issue today. Will let you know when that is resolved.

**Comment by connorlindsey** [comment]
We just released version 0.1.8 of the React package which should fix this. Please try upgrading to the latest version and let us know if you run into any other issues. Thanks! https://www.npmjs.com/package/@knocklabs/react/v/0.1.8
